components/certificate-mgt/org.wso2.carbon.identity.certificate.management/src/test/resources/dbscripts/h2.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE IF NOT EXISTS IDN_CERTIFICATE (
             ID INTEGER NOT NULL AUTO_INCREMENT,
-            UUID CHAR(36) NOT NULL,
+            UUID CHAR(36),
```

Comment:
why this not null is removed ?

Reply:
Removed not null to facilitate the migration requirements where we have to alter the table and add the new column. In such case it will not not work unless we give a default value for the UUID of the existing entries.
Hence we can either remove NOT NULL requirement from the column or have a default value for the existing entries.

Reply:
Since we are avoiding data migrations at this moment proceeding without having the NOT NULL for the UUID column is the most suitable option.

Reply:
ack